### ISSUE TITLE
[BUG] - Single quotes `'` are not supported in `aria` property

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3171]
### Contact Details

Slack: https://porschedev.slack.com/archives/CT7AVHTTQ/p1712671727869439

### JS Framework

All

### Porsche Design System version

3.14.0

### On which device(s) you can see the bug?

All

### In which browsers you can see the bug?

All

### Browser version

All

### Reproduction Example

Stackblitz: https://stackblitz.com/edit/mdbyvm?file=index.html

### Steps to reproduce the behavior

1. Go to https://stackblitz.com/edit/mdbyvm?file=index.html
2. See error (escaped single quote is rendered as double quote)

### Expected behavior

If an escaped single quote `\'` is provided in the `aria` prop string, it should be rendered as such.
It should work with any quote constellation, e.g. `" | ' | ``
